### DISCUSSION TITLE
Update From V2 to V3

### DISCUSSION THREAD

**Discussion by BenOussama180** [Power-Components/livewire-powergrid#457] [Q&A] [answered]
So i followed the update guide from V2 to V3 on powergrid docs, but in my composer.json file doesnt show livewire powergrid on V3, it still in V2

**Comment by marineusde** [comment, marked as answer]
try this in the console:

```
composer require 'power-components/livewire-powergrid:3.0'
```

then the version changed to 3.0x and your composer.json will be updated.
